docs/configure_and_build.rst

```diff
@@ -48,20 +48,29 @@ Windows command line
 --------------------
 Note that CMake is also available on Windows as a GUI (instructions below).  
 This section describes how to use CMake on Windows directly from the command line.   
-Instructions for running CMake through the command line can be found on the :cmake:`CMake site</runningcmake/>` under the heading "Running CMake from the command line". 
+Additional instructions for running CMake through the command line can be found on the :cmake:`CMake site</runningcmake/>` under the heading "Running CMake from the command line".
 
-For Windows only the location of the libXML2 library must be specified through the command line by adding the parameter::
+For Windows we have to have the required libraries zlib and LibXml2 before we can configure and build libCellML.
```

Comment:
`For Windows, we must have the required libraries zlib and libxml2 before...`

(I believe it's `libxml2`, not `LibXml2`, so should update it everywhere. See http://xmlsoft.org/.)

Reply:
e1b924d

Reply:
Still need a comma: `For Windows, we have...`